packages/ckeditor5-dev-build-tools/src/utils.ts

```diff
@@ -60,3 +61,11 @@ export function getUserDependency( name: string ): any {
 
 	return require( path );
 }
+
+/**
+ * `ckeditor5` and `ckeditor5-premium-features` globals.
+ */
+export const CKEDITOR_GLOBALS: GlobalsOption = {
+	ckeditor5: 'CKEDITOR',
+	'ckeditor5-premium-features': 'CKEDITOR_PREMIUM_FEATURES'
+};
```

Comment:
Having this const in utils may be overkill, since we only use it in one place.

Reply:
Yes, it makes sense. Fixed in a71ea6f.